app/src/js/components/Granules/index.js

```diff
@@ -24,12 +24,15 @@ const Granules = ({
   const AllGranulesWithWrapper = withQueryWrapper(AllGranules, queryOptions, setQueryOptions);
 
   function query () {
+    dispatch(listGranules(queryOptions));
+  }
+
+  useEffect(() => {
```

Comment:
I need education here.
is this just dispatching a getCount when the dispatch object changes? what changes the dispatch object?

Reply:
I may have answered my own question, does this mean it only runs once? (like listWorkflows() https://github.com/nasa/cumulus-dashboard/pull/788/files#diff-30c61d87d412853c9930566dd50f2c45R73). 
And that (listworkflows) probably is what's needed in collections/granules.?

Reply:
Right on both counts. Added to collections/granules here https://github.com/nasa/cumulus-dashboard/pull/788/commits/bbf214de2aeb9d0ea7ee8f5d4b1b269c1ac961e9